src/components/expansion-panel/header/styled.tsx

```diff
@@ -0,0 +1,160 @@
+import styled, { css } from 'styled-components';
+import Icon from '../../icon';
+import { theme, Colour } from '@fellesdatakatalog/theme'
+
+interface Props {
+  isExpanded? : boolean;
+  showAlert : boolean;
+}
+
+const CollapseButton = styled.button<Props>`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  font-family: inherit;
+  background-color: ${({ showAlert }) => 
+    showAlert 
+    ? css`${theme.colour(Colour.RED, 'R30')}`
+    : css`${theme.colour(Colour.NEUTRAL, 'N0')}`
+  };
+
+  border: none;
+  padding: 0rem 2.4rem;
```

Comment:
Er det spesiel grunn bak bruken av `rem` istedenfor f.eks. `${theme.spacing('S4')};` ? :)

Reply:
`theme.spacing('S24')` er [hardkodet til 2.4rem](https://github.com/fellesdatakatalog/fdk-kit/blob/develop/packages/theme/src/theme/index.ts). For meg (og etter diskusjon med Jeff) virker det egentlig bare som en mer innviklet måte å skrive 2.4rem på, og da ga det mer mening å gjøre det så enkelt som mulig og skrive kun `2.4rem`. Er mulig det er noe jeg overser her.

Reply:
OK. Ikke super kritisk uansett, siden vi skal over til Figma tokens snart. Men da må vi huske å sjekke (og erstatte) også for rem verdier.